sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProvider.java

```diff
@@ -102,6 +104,12 @@ public static SdkMeterProviderBuilder builder() {
    * }</pre>
    */
   public void registerView(InstrumentSelector selector, AggregatorFactory aggregatorFactory) {
```

Comment:
We should change this to be:

`public void registerView(InstrumentSelector selector, View view)`

Reply:
done